cypress/integration/channel/message_draft_spec.js

```diff
@@ -7,19 +7,19 @@
 // - Use element ID when selecting an element. Create one if none.
 // ***************************************************************
 
-/*eslint max-nested-callbacks: ["error", 3]*/
+/*eslint max-nested-callbacks: ["error", 4]*/
 
 let testTeam;
 
 describe('Message Draft', () => {
     before(() => {
         // # Login as new user
-        cy.loginAsNewUser();
-
-        // # Create new team and visit its URL
-        cy.apiCreateTeam('test-team', 'Test Team').then((response) => {
-            testTeam = response.body;
-            cy.visit(`/${testTeam.name}`);
+        cy.loginAsNewUser().then(() => {
```

Comment:
I don't think having to use `.then()` should be necessary, but maybe I'm missing something.

Reply:
I'm not totally sure about this but let's try and observe.